react/src/components/App.js

```diff
@@ -7,12 +7,12 @@ import Routes from './Routes';
 import ScrollUpButton from './ScrollUpButton';
 import ReactGA from 'react-ga';
 
-const trackingId = "UA-156842946-1";
-ReactGA.initialize(trackingId);
+const trackingId = 'UA-156842946-1';
+ReactGA.initialize(trackingId, { testMode: process.env.NODE_ENV === 'test' });
```

Comment:
Is this actually how you set up a test env? How does it work?

Reply:
So this code basically sets ReactGA into testMode when we're in a testing environment so that tests can actually run on the <App /> component.  Otherwise, it'll throw an error because tests run in a headless browser and it freaks out because it can't initialize.

I got that information from https://github.com/react-ga/react-ga#test-mode